apps/nextra/pages/en/build/guides/your-first-nft.mdx

```diff
@@ -1,768 +1,476 @@
-  recipient: bob.accountAddress,
-});
-const committedTxn = await aptos.signAndSubmitTransaction({
-  signer: alice,
-  transaction: transferTransaction,
-});
-const pendingTxn = await aptos.waitForTransaction({
-  transactionHash: committedTxn.hash,
-});
-```
+You can view these transactions on the Aptos Explorer:
 
-```ts filename="example.ts"
-const alicesDigitalAssetsAfter = await aptos.getOwnedDigitalAssets({
-  ownerAddress: alice.accountAddress,
-  minimumLedgerVersion: BigInt(pendingTxn.version),
-});
-console.log(
-  `Alice's digital assets balance: ${alicesDigitalAssetsAfter.length}`,
-);
+1. Copy the transaction hash from your console
+2. Visit [Aptos Explorer](https://explorer.aptoslabs.com/?network=devnet)
+3. Make sure you're on the correct network (Devnet)
+4. Paste the transaction hash in the search bar
+5. View the details of your transaction, including:
+   - The sender and recipient addresses
+   - The exact time the transaction was processed
+   - Gas fees paid
+   - The digital asset that was transferred
 
-const bobDigitalAssetsAfter = await aptos.getOwnedDigitalAssets({
-  ownerAddress: bob.accountAddress,
-  minimumLedgerVersion: BigInt(pendingTxn.version),
-});
-console.log(`Bob's digital assets balance: ${bobDigitalAssetsAfter.length}`);
-```
+This is a great way to verify your transactions and understand how they're recorded on the blockchain.
+</Steps>
 
-  </Tabs.Tab>
-  <Tabs.Tab>
-
-```python filename="example.py"
-print("\n=== Transferring the token to Bob ===")
-txn_hash = await token_client.transfer_token(
-    alice,
-    token_addr,
-    bob.address(),
-)
-await rest_client.wait_for_transaction(txn_hash)
-```
+## Summary of the Key Steps
 
-```python filename="example.py"
-async def transfer_token(
-    self, owner: Account, token: AccountAddress, to: AccountAddress
-) -> str:
-    return await self.client.transfer_object(owner, token, to)
-```
+1. **Setup Your Project**: Initialize a Node.js project and install `@aptos-labs/ts-sdk` and `dotenv`.
+2. **Configure Your Environment**: Set `APTOS_NETWORK` in a `.env` file.
+3. **Configure TypeScript**: Create a proper `tsconfig.json` file to ensure type safety.
+4. **Initialize the `Aptos` Client**: Connect to Aptos using `new Aptos(new AptosConfig({ network }))`.
+5. **Create & Fund Accounts**: Generate and fund accounts on devnet to pay for transactions.
+6. **Create a Collection:** All Digital Assets need a collection to own them.
+7. **Mint a Digital Asset**: Create custom Digital Assets which point to whichever `uri` you choose.
+8. **Transfer the Asset**: Move the digital asset from one account to another.
+9. **Verify the Result**: Confirm that the recipient now holds the newly minted digital asset.
+10. **Log Transaction Hashes**: Output all transaction hashes for easy reference and exploration.
+11. **Explore Transactions**: View your transactions on the Aptos Explorer to understand how they're recorded on-chain.
```

Comment:
Maybe remove or move to the top? The sidebar already has a table of contents and I'm not sure why a summary would be at the end.